datadog/resource_datadog_child_organization.go

```diff
@@ -0,0 +1,112 @@
+package datadog
+
+import (
+	datadogV1 "github.com/DataDog/datadog-api-client-go/api/v1/datadog"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceDatadogChildOrganisation() *schema.Resource {
+	return &schema.Resource{
+		Description: "Provides a Datadog Organization resource. This can be used to create and manage Datadog organizations.",
+		Create:      resourceDatadogChildOrganizationCreate,
+		Read:        resourceDatadogChildOrganizationRead,
+		Update:      resourceDatadogChildOrganizationUpdate,
+		Delete:      resourceDatadogChildOrganizationDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceDatadogOrganizationImport,
```

Comment:
Update this function name to be consistent with the other function names.

```suggestion
			State: resourceDatadogChildOrganizationImport,
```